spec/CloudCode.Validator.spec.js

```diff
@@ -860,6 +860,60 @@ describe('cloud validator', () => {
       });
   });
 
+  it('basic validator requireUserRole', async function (done) {
+    Parse.Cloud.define(
+      'cloudFunction',
+      () => {
+        return true;
+      },
+      {
+        requireUser: true,
+        requireUserRole: ['Admin'],
+      }
```

Comment:
DX: What happens now if the developer only provides `requireUserRole` may you should automatically set  `requireUser ` to true ?